operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,502 @@
+    -1         48.90472  root default
+   -19         13.97278      rack x3000
+    -7         13.97278          host ncn-s001
+     3    ssd   1.74660              osd.3             up   1.00000  1.00000
+     7    ssd   1.74660              osd.7             up   1.00000  1.00000
+    11    ssd   1.74660              osd.11            up   1.00000  1.00000
+    14    ssd   1.74660              osd.14            up   1.00000  1.00000
+    17    ssd   1.74660              osd.17            up   1.00000  1.00000
+    20    ssd   1.74660              osd.20            up   1.00000  1.00000
+    22    ssd   1.74660              osd.22            up   1.00000  1.00000
+    25    ssd   1.74660              osd.25            up   1.00000  1.00000
+   -21         17.46597      rack x3001
+   -11         13.97278          host ncn-s003
+     4    ssd   1.74660              osd.4             up   1.00000  1.00000
+     9    ssd   1.74660              osd.9             up   1.00000  1.00000
+    12    ssd   1.74660              osd.12            up   1.00000  1.00000
+    15    ssd   1.74660              osd.15            up   1.00000  1.00000
+    18    ssd   1.74660              osd.18            up   1.00000  1.00000
+    21    ssd   1.74660              osd.21            up   1.00000  1.00000
+    24    ssd   1.74660              osd.24            up   1.00000  1.00000
+    27    ssd   1.74660              osd.27            up   1.00000  1.00000
+    -3          3.49319          host ncn-s004
+     0    ssd   1.74660              osd.0             up   1.00000  1.00000
+     5    ssd   1.74660              osd.5             up   1.00000  1.00000
+   -23         17.46597      rack x3002
+    -9         13.97278          host ncn-s002
+     2    ssd   1.74660              osd.2             up   1.00000  1.00000
+     8    ssd   1.74660              osd.8             up   1.00000  1.00000
+    10    ssd   1.74660              osd.10            up   1.00000  1.00000
+    13    ssd   1.74660              osd.13            up   1.00000  1.00000
+    16    ssd   1.74660              osd.16            up   1.00000  1.00000
+    19    ssd   1.74660              osd.19            up   1.00000  1.00000
+    23    ssd   1.74660              osd.23            up   1.00000  1.00000
+    26    ssd   1.74660              osd.26            up   1.00000  1.00000
+    -5          3.49319          host ncn-s005
+     1    ssd   1.74660              osd.1             up   1.00000  1.00000
+     6    ssd   1.74660              osd.6             up   1.00000  1.00000
+   ```
+
+### 4 Resiliency Monitoring Service (RMS)
+
+To monitor and debug RMS, check the logs of the `cray-rrs` Kubernetes pod running in the `rack-resiliency` namespace. 
+
+Follow the steps below:
+
+Checking logs for `cray-rrs*` pod in `rack-resiliency` namespace
+
+**Step-by-Step Procedure**
+
+1. (`ncn-mw#`) List the pods in the `rack-resiliency` namespace to identify the name of the `cray-rrs` pod

```

Comment:
If the only purpose of this is to get the pod name, you can just give them a single command to do it, where they don't need to pick it from the output. You could put it into a variable so you can supply it for the next command that way. 
That is more user friendly for an administrator following the procedure.

Something like

```bash
RRS_POD=$(kubectl get pods -n rack-resiliency -l app.kubernetes.io/instance=cray-rrs -o custom-columns=:.metadata.name --no-headers); echo "${RRS_POD}"
```

We put the `echo` at the end so they can confirm that it actually got a pod name, and wasn't just empty output (which would only be the case if the pod was missing entirely, or the kubectl command was having other issues).